Sources/TuistGenerator/Settings/DefaultSettingsProvider.swift

```diff
@@ -19,6 +19,8 @@ public protocol DefaultSettingsProviding {
 }
 
 public final class DefaultSettingsProvider: DefaultSettingsProviding {
+    private let defaultSwiftVersion = "5.0"
```

Comment:
nit: as it's used only once, we could even just use it when needed

Reply:
makes sense 👍  

I've inlined it for the moment, we can revise this as / when needed